### PR TITLE
#232 Create a jar from classes folder that can be scanned

### PR DESCRIPTION
**Pull request by jperedadnr**
for user’s configuration files

### CODE REVIEW

File: src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java [hunk 1]
```diff
@@ -114,6 +114,7 @@ public boolean compile(String cp) throws IOException, InterruptedException {
         }
         Path gvmPath = paths.getGvmPath();
         FileOps.rmdir(paths.getTmpPath());
+        Files.createDirectories(paths.getTmpPath());
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java]: Don't we have generic code that makes sure the required paths are created?

File: src/main/java/com/gluonhq/substrate/config/ConfigResolver.java [hunk 2]
```diff
@@ -61,11 +64,35 @@
 
     private final List<File> jars;
 
-    public ConfigResolver(String classpath) {
-        this.jars = new ClassPath(classpath)
+    /**
+     * ConfigResolver constructor
+     *
+     * @param classpath a string with the full classpath of the user's project
+     * @param tmpPath a temporal path where classes.jar will be added
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/config/ConfigResolver.java]: is tmpPath needed outside of this method? If not, it seems better to create/delete it inside this method?

File: src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java (hunk 1 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java]: We do, but the line before just removes that path

File: src/main/java/com/gluonhq/substrate/config/ConfigResolver.java (hunk 2 above)
Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/config/ConfigResolver.java]: We just need a temporal path, and using `client/archOS/gvm/tmp` seems convenient, but then we need to pass that path.

Comment (johanvos) [reply src/main/java/com/gluonhq/substrate/config/ConfigResolver.java]: Don't we have Paths.getTmpPath or so?

Comment (jperedadnr) [reply src/main/java/com/gluonhq/substrate/config/ConfigResolver.java]: No, we don't. Well, we could use Files.createTempDirectory, outside client/archOS/gvm

Comment (johanvos) [reply src/main/java/com/gluonhq/substrate/config/ConfigResolver.java]: If we don't need the results outside of this method, Files.createTempDirectory sounds better.

**Pull request merged**